Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -534,6 +535,8 @@ static inline void continueAfterRequest(RetainPtr<id <ASCCredentialProtocol>> cr
 
 static inline bool canCurrentProcessAccessPasskeysForRelyingParty(const WebCore::SecurityOriginData& data)
 {
+    if (os_variant_is_basesystem("WebKit"))
```

Comment:
Just like in Web content, it's better to do feature detection, not environment detection. Normally, we just check if the framework can be loaded, using `SOFT_LINK_OPTIONAL`.